File: 백준/Bronze/32498. Call for Problems/Call for Problems.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)

	count := 0

	for i := 0; i < n; i++ {
		var d int
		fmt.Scan(&d)
		if d%2 != 0 {
			count++
		}
	}

	fmt.Println(count)
}

File: 백준/Bronze/32498. Call for Problems/Call for Problems.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());
    let mut input = reader.lines();

    if let Some(Ok(line)) = input.next() {
        let n = line.parse::<u32>().unwrap();
        let mut count = 0;
        for i in 0..n {
            if let Some(Ok(line)) = input.next() {
                let n = line.parse::<u32>().unwrap();
                if n % 2 != 0 {
                    count += 1;
                }
            }
        }
        writeln!(writer, "{}", count).unwrap();

    }

    writer.flush().unwrap();
}
